packages/@markuplint/config-presets/src/preset.html-standard.json

```diff
@@ -124,6 +124,21 @@
 				"require-accessible-name": false
 			}
 		},
+		{
+			/**
+			 * Require `title` attr in `<input pattern>`
+			 *
+			 * When an `<input>` element has a `pattern` attribute specified, authors should include a `title` attribute to give a description of the pattern.
+			 *
+			 * @see https://html.spec.whatwg.org/multipage/grouping-content.html#the-figure-element
```

Comment:
Can anybody from the team confirm this URL is really what you wanted to use here?